content/en/blog/_posts/XXXXXX-kubernetes-v1-35-sneak-peek.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: blog
+title: 'Kubernetes v1.35: Sneak peek'
```

Comment:
to avoid audience thinking "Sneak Peek" is the name of the release 😅 
```suggestion
title: 'Kubernetes v1.35 Sneak Peek'
```
we might need to carry this fix to the handbook template